pyaedt/generic/plot.py

```diff
@@ -1125,7 +1125,7 @@ def objects(self):
 
     @pyaedt_function_handler()
     def add_object(self, cad_path, cad_color="dodgerblue", opacity=1, units="mm"):
-        """Add an mesh file to the scenario. It can be obj or any of pyvista supported files.
+        """Add a mesh file to the scenario. It can be obj or any of pyvista supported files.
```

Comment:
```suggestion
        """Add a mesh file to the scenario.
        
        The mesh file can be an object or any of the PyVista supported files.
```
The need for a brief desription is obvious if you ever switch to [Sphinx AutoAPI](https://pypi.org/project/sphinx-autoapi/) extension. It generates summary tables and longer descriptions of Python objects get truncated.